modules/cluster-autoscaler-cr.adoc

```diff
@@ -48,16 +48,19 @@ you assign to each pod.
 <4> Specify the maximum number of cores to deploy.
 <5> Specify the minimum amount of memory, in GiB, per node.
 <6> Specify the maximum amount of memory, in GiB, per node.
-<7> Specify the type of GPU node to deploy. Only `nvidia.com/gpu` and `amd.com/gpu`
-are valid types.
+<7> Optionally, specify the type of GPU node to deploy. Only
+`nvidia.com/gpu` and `amd.com/gpu` are valid types.
 <8> Specify the minimum number of GPUs to deploy.
 <9> Specify the maximum number of GPUs to deploy.
 <10> Specify whether the ClusterAutoscaler can remove unnecessary nodes.
-<11> Specify the period, in minutes, to wait before deleting a node after
-a node has recently been _added_.
+<11> Optionally, specify the period, in minutes, to wait before deleting a node after
+a node has recently been _added_. If you do not specify a value, the default
+value of `10m` is used.
 <12> Specify the period, in minutes, to wait before deleting a node after
-a node has recently been _deleted_.
+a node has recently been _deleted_. If you do not specify a value, the default
+value of `10s` is used.
 <13> Specify the period, in seconds, to wait before deleting a node after
-a scale down failure occured.
+a scale down failure occurred. If you do not specify a value, the default
+value of `3m` is used.
 <14> Specify the period, in seconds, before an unnecessary node is eligible
```

Comment:
What's the default value for `unneededTime`?

Reply:
Default is 10 minutes.